src/System.Windows.Forms/src/System/Windows/Forms/Controls/TreeView/TreeNode.cs

```diff
@@ -1250,20 +1250,20 @@ private void SortChildren(TreeView? parentTreeView)
                         continue;
                     }
 
-                    if (sorter.Compare(_childNodes[j] /*previous*/, _childNodes[min] /*current*/) <= 0)
+                    if (sorter.Compare(_children[j] /*previous*/, _children[min] /*current*/) <= 0)
```

Comment:
```suggestion
                    if (sorter.Compare(previous: _children[j], current: _children[min]) <= 0)
```
(if these are indeed the parameter names)

Reply:
actually they are not.